### DISCUSSION TITLE
Max view option

### DISCUSSION THREAD

**Discussion by leoap** [t1m0n/air-datepicker#630] [Announcements]
Hello!
I've started to use Air Datepicker and it is great, thank you for the efforts of maintaining it.

I was wondering if there is any way to limit the user to view just the 'months' and 'days', so he would not be able choose 'years'. 
I understand that creating a Date object requires a year, but in that case, the year may be the current year.

Thanks again!

**Comment by t1m0n** [comment]
Hello! Thank you for your feedback!
You could do something like this

```javascript
let dp = new Datepicker('myElement', {
    onChangeView(view) {
        if (view === 'years') {
            dp.setCurrentView('months');
        }
    }
});
```
